build/docker-exec

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [[ $# -ne 2 ]] ; then
+    echo "build/docker-exec - run a rake command on a specified ruby version"
+    echo "Usage:"
+    echo "  build/docker-exec 2.2.5 spec"
+    exit 1
+fi
+
+ruby_version="$1"
+rake_task="$2"
+
+cp -r /app /src
+cd /src/app || exit 1
```

Comment:
Yeah. For now I didn't want to deal with the back pressure of making the commands not write to the system.